### ISSUE TITLE
[Question] How auto-mount work and can I block-list my Linux root FS?

### ISSUE THREAD

**Issue by Mart-Bogdan** [maharmstone/btrfs#626]
I have read in readme about automount and paragon, more specific:
> * The drive doesn't show up and Paragon software has been installed
Paragon's filesystem-reading software is known to disable automount. 
Disable or uninstall Paragon, then re-enable automount by running diskpart and typing automount enable.

Is it not possible to assign drive letter with **Disk Management** in windows, like how I can change/add drive letters for NTFS/FAT partitions? I could even add directory mount-points there.

To be more specific what I want is to not automount btrfs partitions, but be able to manually mount them on windows. Having sub-volume as mount is a plus. (as I understand it's registry option).

But as I understand without aoutomount I won't be able to use USB drives? (with ntfs/fat). But I guess I could live with it, if I'll have to use MMC or console to mount USB drives.

What I want to do is to create partition that would be used as share between widnows and Linux and also to hold data from windows.

My current work requires me to store huge volumes of data and being abele to use cp --reflink. It's gamedev and it's huge volume, and having multiple copies of different branches require impractical amount of time. Since ReFS is not available anymore, I decided to look in direction of winbtrfs. 

But I don't want to risk my root/home partitions, as my whole life is there. I want this driver to be used only with single partition that I can afford to lose. 

To sum up my questions:

* Is it possible to assign drive letters(folder mount points) to BTRFS with Drive Manager or only via automount/registry?
* Is it possible to forbade mounting specific file-system (it spans two parttitions now)?
* Would above interfere with using external USB storage? 

**P.S.** I understand that it is not an issue/bug, but wasn't able to find place for discussions in readme, or GH discussions enabled. And I decided it better to be public instead of email, in case it would be useful for others.  

Thanks.

**Comment by Mart-Bogdan** [comment]
Can someone answer here, please?

**Comment by maharmstone** [comment]
Yes, it's known that Disk Management doesn't play nicely with Btrfs volumes. The shell extension allows you to change drive letters if you need to.

The README tells you how to exclude certain volumes - search for the word "Ignore".
